src/services/sessionInvite.js

```diff
@@ -296,8 +323,16 @@ module.exports = class UserInviteHelper {
 						}
 					}
 					//Default Validation
-					const validateDefaultBBB = () => {
-						setMeetingInfo(common.MEETING_VALUES.BBB_LABEL, common.BBB_VALUE)
+					const validateDefaultBBB = async () => {
+						if (process.env.DEFAULT_MEETING_SERVICE === common.BBB_VALUE) {
+							setMeetingInfo('', '')
```

Comment:
@sumanvpacewisdom its common statement for both condition keep it outside of if 
setMeetingInfo('', '')